src/services/api/streaming.ts

```diff
@@ -78,6 +67,11 @@ export class StreamingAPI extends BaseYdbAPI {
         });
 
         if (!response.ok) {
+            const responseData = await response.json().catch(() => ({}));
```

Comment:
Why do you add empty `.catch(()=>{})` here?

Reply:
When we call response.json(), it attempts to parse the response body as JSON
If the response body is not valid JSON, response.json() will throw an error
The .catch(() => ({})) catches this error and returns an empty object as a fallback
This ensures that even if the response is not JSON, we don't throw an error but instead get an empty object

Reply:
Below is throw new Error(`${response.status}`);

I believe response should have status even if response body is not JSON